WordPress/Classes/ViewRelated/NUX/SigninEditingState.swift

```diff
@@ -1,6 +1,6 @@
 import Foundation
 
 class SigninEditingState {
-    static var signinEditingStateActive = false
+    static var signinEditingStateActive = true
```

Comment:
Let's do this in a separate PR.  A quick pass through the wpcom password flow has the keyboard jumping up and down between navigation and it doesn't look too good.  I think its going to need more attention than this one line change.